### PR TITLE
Fix build failure due to new linter rule

### PR DESCRIPTION
**Pull request by alrusdi**
I've broken master branch by blindly accepting some approved PRs
Below are fixes for the problems i found so far

### CODE REVIEW

**Review by kberg — CHANGES_REQUESTED** [review]
Did you mean to include make_source.py?

**Comment by alrusdi** [comment]
> Did you mean to include make_source.py?

no-no! It is mistake of cource

**Comment by bafolts** [comment]
Trying to think of the most painless way to introduce these linting rules. I feel there is enough value with using `let` and `const` versus `var` that the pain is worth it. I thought I had this rule turned on initially. This should have covered both `src/Player.ts` and `src/Game.ts` which get a lot of modifications as those files are very large. I will do the change per directory hoping to avoid these continual conflicts. Sorry for any pain in the meantime.

**Comment by alrusdi** [comment]
> Trying to think of the most painless way to introduce these linting rules. 
We need them anyway. Clean code is better code!

File: src/locales/initial_source.json [hunk 1]
```diff
@@ -0,0 +1,10554 @@
+[
+    {
+        "model": "translations.source",
+        "pk": 1,
+        "fields": {
```
Comment (bafolts) [inline src/locales/initial_source.json:5]: This file also seems unrelated to fixing linting rule failure.

**Comment by kberg** [comment]
> > Trying to think of the most painless way to introduce these linting rules.
> 
> We need them anyway. Clean code is better code!

Strongly agree.

File: src/locales/initial_source.json (hunk 1 above)
Comment (alrusdi) [reply src/locales/initial_source.json]: Yes, it is fixed too

**Review by bafolts — APPROVED** [review]

**Review by kberg — APPROVED** [review]

**Pull request merged**